### PR TITLE
Features/add owner repo input

### PR DESCRIPTION
**Pull request by Rasmus-Rosted**
This implementation gives the user the possibility to specify the owner and repository where the branches that should be deleted are located. This is done in order to not **only** be able to delete branches in the repository that the action is called from or the GitHub token dictates.

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -9,6 +9,12 @@ inputs:
     description: GitHub token
     required: false
     default: ${{github.token}}
+  owner:
+    description: Owner of the repository. Will be deducted from github_token if it is not provided
+    required: false
+  repository:
+    description: The repository containing the branch(es) to be deleted. Will be deducted from github_token if it is not provided
```
Comment (MindaugasLaganeckas) [inline action.yml]: Repository name will be deducted from env vars, not from github_token

File: main.js [hunk 2]
```diff
@@ -5,10 +5,12 @@ async function main() {
     try {
         const token = core.getInput("github_token", { required: true })
         const numbers = core.getInput("numbers")
+        const owner = core.getInput("owner")
+        const repository = core.getInput("repository")
         const branches = core.getInput("branches")
         const prefix = core.getInput("prefix")
         const suffix = core.getInput("suffix")
-
+
```
Comment (MindaugasLaganeckas) [inline main.js]: Do not introduce unnecessary changes.

**Review by MindaugasLaganeckas — CHANGES_REQUESTED** [review]
Nice!

File: main.js (hunk 2 above)
Comment (Rasmus-Rosted) [reply main.js]: Indentation has been removed

File: action.yml (hunk 1 above)
Comment (Rasmus-Rosted) [reply action.yml]: Let's talk about this when we both have time

Comment (Rasmus-Rosted) [reply action.yml]: Description changed for both 'owner' and 'repository'
